app/src/main/java/org/fossasia/susi/ai/helper/CredentialHelper.kt

```diff
@@ -16,7 +16,7 @@ import java.util.regex.Pattern
  */
 object CredentialHelper {
 
-    private val PASSWORD_PATTERN = Pattern.compile("^.{6,64}$")
+    private val PASSWORD_PATTERN = Pattern.compile("^(?=.*[a-z])(?=.*[@#\$%]).{8,64}\$")
```

Comment:
Add tests

Reply:
Can you please explain what you wanted to say through this statement.

Reply:
I mean add tests for this logic